scripts/install-systemd-multi-user.sh

```diff
@@ -9,6 +9,38 @@ readonly SERVICE_DEST=/etc/systemd/system/nix-daemon.service
 readonly SOCKET_SRC=/lib/systemd/system/nix-daemon.socket
 readonly SOCKET_DEST=/etc/systemd/system/nix-daemon.socket
 
+
+# Path for the systemd override unit file to contain the proxy settings
+readonly SERVICE_OVERRIDE=${SERVICE_DEST}.d/override.conf
+
+create_systemd_override() {
+     header "Configuring proxy for the nix-daemon service"
+    _sudo "create directory for systemd unit override" mkdir -p "$(dirname $SERVICE_OVERRIDE)"
+    cat <<EOF | _sudo "create systemd unit override" tee "$SERVICE_OVERRIDE"
+[Service]
+$1
+EOF
+}
+
+# Gather all non-empty proxy environment variables into a string
+create_systemd_proxy_env() {
+    vars="http_proxy https_proxy ftp_proxy no_proxy HTTP_PROXY HTTPS_PROXY FTP_PROXY NO_PROXY"
+    for v in $vars; do
+        if [ -n "${!v}" ]; then
```

Comment:
This yields an error:

```
ubuntu-18-10-install    ./nix/install-systemd-multi-user.sh: line 29: !v: unbound variable
```

and the daemon install breaks on this.

I've been so slow to review this. I'm working on patching this myself in conjunction with a documentation patch.

Reply:
@grahamc  Thanks for catching the issue and fixing it. For some reason, the issue did not show up in my tests on ubuntu 18.04.